### PR TITLE
Skip git manifests deletion

### PR DESCRIPTION
**Pull request by invin12**
Optionally skip manifests deletion from git repository on terraform destroy

### CODE REVIEW

**Comment by swade1987** [comment]
@invin12, what will be the default behaviour?

**Comment by invin12** [comment]
> @invin12, what will be the default behaviour?

By default, the provider will delete all Flux manifests from git(current behavior).
My PR add an option to skip this by leaving manifests in repository after Flux uninstall(terraform destroy).

**Comment by swade1987** [comment]
Sounds good @invin12 could you please resolve [this](https://github.com/fluxcd/terraform-provider-flux/pull/595/checks?check_run_id=21115870817).

**Comment by invin12** [comment]
> Sounds good @invin12 could you please resolve [this](https://github.com/fluxcd/terraform-provider-flux/pull/595/checks?check_run_id=21115870817).

Done

File: docs/resources/bootstrap_git.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ resource "flux_bootstrap_git" "this" {
 - `components` (Set of String) Toolkit components to include in the install manifests. Defaults to `[source-controller kustomize-controller helm-controller notification-controller]`
 - `components_extra` (Set of String) List of extra components to include in the install manifests.
 - `disable_secret_creation` (Boolean) Use the existing secret for flux controller and don't create one from bootstrap
+- `delete_git_manifests` (Boolean) Delete manifests from git repository.
```
Comment (swade1987) [inline docs/resources/bootstrap_git.md:33]: Can you please add a `Defaults to true` at the end?

Comment (invin12) [reply docs/resources/bootstrap_git.md]: Done. But defaults to `false`) - don't delete git manifests(current provider behavior).

**Comment by swade1987** [comment]
Can you also rebase as the branch is out of date.

**Comment by invin12** [comment]
> Can you also rebase as the branch is out of date.

done

**Pull request closed without merging**